tests/foreman/cli/test_contentviews.py

```diff
@@ -1479,22 +1366,25 @@ def test_cv_publish_version_changes_in_target_env(self):
             self.fail(err)
 
         # Associate repo to CV
-        result = ContentView.add_repository({u'id': new_cv['id'],
-                                             u'repository-id': new_repo['id']})
-        self.assertEqual(result.return_code, 0,
-                         "Repo was not associated to selected CV")
-        self.assertEqual(len(result.stderr), 0, "No error was expected")
+        result = ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'repository-id': new_repo['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
 
         # Publish a version1 of CV
         result = ContentView.publish({u'id': new_cv['id']})
-        self.assertEqual(result.return_code, 0,
-                         "Publishing version1 of CV was not successful")
-        self.assertEqual(len(result.stderr), 0, "No error was expected")
+        self.assertEqual(
+            result.return_code, 0,
+            "Publishing version1 of CV was not successful"
```

Comment:
Yes, we have two versions being created on this test, and I decided to leave this message to make easier to know if the first or second version that failed.